src/filter/filter.tsx

```diff
@@ -127,23 +137,31 @@ const FilterBase = ({
                     {toggleFilterButtonLabel}
                 </FilterButton>
                 <Overlay show={visible} disableTransition>
-                    <MobileOverlayContainer
-                        data-id="filter-mobile"
-                        data-testid="filter-mobile"
+                    <FloatingFocusManager
+                        context={context}
+                        modal={true}
+                        initialFocus={-1}
                     >
-                        <MobileContainer ref={mobileNodeRef}>
-                            {renderHeader("mobile")}
-                            <FilterBody>{renderChildren("mobile")}</FilterBody>
-                            <FilterFooter>
-                                <FilterDoneButton
-                                    onClick={handleDoneClick}
-                                    type="button"
-                                >
-                                    {doneButtonLabel}
-                                </FilterDoneButton>
-                            </FilterFooter>
-                        </MobileContainer>
-                    </MobileOverlayContainer>
+                        <MobileOverlayContainer
+                            data-id="filter-mobile"
```

Comment:
need to set this so that the element is registered with Floating UI to perform the focus trap

```
ref={refs.setFloating}
```